xls/dslx/stdlib/std.x

```diff
@@ -177,6 +177,53 @@ fn iterative_div_test () {
   ()
 }
 
+// Binary Division algorithm as outlined by:
+// http://www.dragonwins.com/domains/GetTechEd/de248/binary_division.htm
+fn iterative_binary_div<N: u32, DN: u32 = {N * u32:2}>(dividend: bits[N], divisor: bits[N]) -> (bits[N], bits[N]) {
```

Comment:
May want a comment describing return values (and their order)

Reply:
I believe this was addressed in https://github.com/google/xls/pull/932